bin/ttl-to-json

```diff
@@ -145,7 +146,7 @@ Promise.all(readFiles).then(fileContent => {
      // from the original json input on a round trip translation. 
      var filterBnodeAttrs = [ 'id', '_id', '@id' ];
 
-     ttlJsonLd.rdfToJsonLd(graph, frameText, filterBnodeAttrs).then(json => { 
+     TtlJsonLd.rdfToJsonLd(graph, frameText, filterBnodeAttrs).then(json => { 
```

Comment:
I like this convention of using capital letters to indicate imported objects, but there is a (not necessarily contradictory) convention to use it for class names . Whatever the conventions we go with, let's try to document them in the team-docs repo -- maybe a CONTRIBUTING.md file